### ISSUE TITLE
Rename WordCount to make it clear that it is an example program.

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/hellbender#537]
Any thoughts on names? `ExampleCommandLineDataflowProgram`

**Comment by pgrosu** [comment]
I like descriptive names, but probably in this instance just updating the [`usage` and `usageShort`](https://github.com/broadinstitute/hellbender/blob/master/src/main/java/org/broadinstitute/hellbender/tools/dataflow/WordCount.java#L15-16) description lines to something like this might be sufficient - otherwise the name might be too long during regular usage:

```
        usage = "This is a test program for Dataflow via the command-line to count the usages of every word in a text",
        usageShort = "Dataflow CLI test program for counting words",
```

I might even got as far as shortening the name to `wc`, since I'm sure when used often during testing could bring happiness :)

~p

**Comment by akiezun** [comment]
@edwardkw for you

**Comment by lbergelson** [comment]
closed by #594 

Thanks @pgrosu for the suggested text.
